src/components/Dialog/ProductTransferDialog.vue

```diff
@@ -42,7 +42,7 @@ const store = useStore();
 const emits = defineEmits(["close-dialog"]);
 
 const allUsers = computed(() => store.getters["users/allUsers"]);
-const userOptions = allUsers.value.map((user) => user.name);
+const userOptions = allUsers.value.map((user) => user.firstName);
```

Comment:
First Name is not unique and therefore couldn't be used for user identifying. First Name Last Name wouldn't be necessarily unique as well. We should go for email.

Reply:
done